src/ExtensionsSample/Samples/DocumentDBTriggerSamples.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json.Linq;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the DocumentDB samples:
+    // 1. Create a new DocumentDB account
+    // 2. Add the DocumentDB Connection String to the 'AzureWebJobsDocumentDBTriggerConnectionString' App Setting in app.config    
+    // 3. Create a Database called ItemDb.
+    // 4. Create a Collection named ItemCollection. This will be the monitored collection.
+    // 5. Create a Collection named Leases. This will be used to store auxiliary leases to support multi-observer scenarios and partitioning.
+    // 6. Optionally create a Collection named ItemCollectionCopy for the ListenAndCopy scenario. And add an 'AzureWebJobsDocumentDBConnectionString' App Setting in app.config with its Connection String
+    // 7. Add typeof(DocumentDBTriggerSamples) to the SamplesTypeLocator in Program.cs
+    public static class DocumentDBTriggerSamples
+    {
+        // Sample implementation of the CosmosDBTrigger that listens for changes in a collection.
+        // The trigger uses an auxiliary collection for leases for multiple partitions.
+        public static void Listen(
+            [DocumentDBTrigger("ItemDb", "ItemCollection", LeaseDatabaseName = "ItemDb", LeaseCollectionName = "Leases")] IReadOnlyList<Document> modifiedDocuments,
+            TraceWriter log)
+        {
+            foreach (Document modifiedDocument in modifiedDocuments)
+            {
+                log.Info(modifiedDocument.Id);
```

Comment:
No, according to the [docs](https://docs.microsoft.com/en-us/azure/cosmos-db/change-feed#how-change-feed-works-in-azure-cosmos-db) what it gets published are the documents that are either inserted or modified. Deletions are not published. It contains just the document, not the reason of why it was published in the feed.